src/shared/parameter_v2/scripts/dynamic_parameter_schema.py

```diff
@@ -0,0 +1,144 @@
+SUPPORTED_TYPES = [
+    "bool",
+    "int",
+    "unsigned",
+    "double",
+    "std::string",
+    "factory",
+    "enum",
+]
+
+INCLUDE_DEF_SCHEMA = {
+    "type": "array",
+    "items": [{"type": "string"}, {"additionalProperties": False}],
+    "uniqueItems": True,
+    "minItems": 1,
+}
+
+PARAM_DEF_SCHEMA = {
+    "type": "array",
+    "uniqueItems": True,
+    "minItems": 1,
```

Comment:
Is there any way to verify what happens if the .yaml file doesn't meet the `minItems` of 1 requirement?

Reply:
the schema is strict, so if you have an 

```
include: 
```
with no includes, you will see the jsonschema validation error

only
```
include: 
- "foo.yaml"
```
and more are valid

Reply:
I was going to add tests for it, but this is how the schema is meant to be used

we will implicitly test everything w/ the overall tests for the generated stuff